cmd/dlv/cmds/commands.go

```diff
@@ -358,6 +358,8 @@ func execute(attachPid int, processArgs []string, conf *config.Config) int {
 
 	var status int
 	if Headless {
+		// Print listener address
+		fmt.Printf("start listener addr %s\n", listener.Addr())
```

Comment:
This should be a `log.Printf`, and lets change the wording to "API server listening at: %s\n".